CC-14555 Add IG for BAPI and BAPI Merchant Relationships. (#693)
spryker/spryker-docs: docs/scos/dev/feature-integration-guides/202200.0/back-end-api-for-merchant-relationships-integration-guide.md

---

> title: Back End API for Merchant Relationships feature integration

Reviewer: ```suggestion
title: Backend API for Merchant Relationships feature integration
```

> description: Integrate the Back End API for Merchant Relationships into your project

Reviewer: ```suggestion
description: Integrate the Backend API for Merchant Relationships into your project
```

template: feature-integration-guide-template
---

> # Back End API for Merchant Relationships feature integration

Reviewer: ```suggestion
# Backend API for Merchant Relationships feature integration
```

This document describes how to integrate the [BAPI Merchant Relationships]({link to a respective feature overview}) into a Spryker project.

## Install feature core

> Follow the steps below to install the {Back End API for Merchant Relationships feature core.

Reviewer: ```suggestion
Follow the steps below to install the {Backend API for Merchant Relationships feature core.
```

### Prerequisites

To start feature integration, integrate the required features and Glue APIs:

| NAME                              | VERSION           | INTEGRATION GUIDE                                                                                                                                                            |
|-----------------------------------|-------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|

> | Back End API                      | {{page.version}}  | [Back End API feature integration](/docs/scos/dev/feature-integration-guides/{{page.version}}/back-end-api-feature-integration.html)                                                                         |

Reviewer: ```suggestion
| Backend API                      | {{page.version}}  | [Backend API feature integration](/docs/scos/dev/feature-integration-guides/{{page.version}}/back-end-api-feature-integration.html)                                                                         |
```

Reviewer: I think the URL should also be backend-api-feature-integration (instead of back-end), but I don't know where that URL came from/if we can change it.
